### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by JaredDavis22** [tomj74/chunk-templates#40] [Q&A] [answered]
Hi,

Hope all is well.

Where are the license details?

I see "Chunk is free and open-source software by Tom McClure. Apache License 2.0."  on https://www.x5software.com/chunk/examples/ChunkExample?loc=en_US

In Chunk.java this is in the code:
// Project Title: Chunk
// Description: Template Engine
// Copyright: Waived. Use freely.
// Author: Tom McClure

I anticipate some issues for the legal review without a license.txt in github.

Would it be possible to dump an apache license into the repo?

Thanks again.

Jared

**Comment by tomj74** [comment, marked as answer]
Switching to MIT license. added LICENSE file to root of repo.
